### ISSUE TITLE
'Menu Top Links Colour' and 'Dropdown Background' colour do not appear to affect menu colours.

### ISSUE THREAD

**Issue by deskspace** [unetics/ECHO#2]
Unless I've misunderstood what it is supposed to affect exactly, the 'Menu Top Links Colour' selection in Nav Options does not appear to affect the colour of the links in the dropdown menu, nor does the 'Dropdown Background' colour.

**Comment by unetics** [comment]
yeah sorry about that i added the 'Dropdown Background' option but didn't link to the css compiler on the todo list
